app.py

```diff
@@ -281,12 +281,7 @@ def fetch_images(image_repo, recommended_articles):
             newsletter_id = request.args.get("newsletter_id")
             feedbackType = request.args.get("feedbackType")
 
-            if feedbackType == "positive":
-                ispositive = True
-            else:
-                ispositive = False
-
-            newsletter_repo.store_newsletter_feedback(account_id, newsletter_id, ispositive)
+            newsletter_repo.store_newsletter_feedback(account_id, newsletter_id, feedbackType)
```

Comment:
I just wrapped my head around this change.

Are we changing feedback from boolean to string?